Monika After Story/game/script-topics.rpy

```diff
@@ -2353,7 +2353,22 @@ label monika_rain_holdme:
 
         call monika_holdme_end
         # small affection increase so people don't farm affection with this one.
-        $ mas_gainAffection(modifier=0.25)
+        python:
+            # Monika likes being held for long periods of time, the longer the better
+            mod = 0.25
+            if elapsed_time <= datetime.timedelta(minutes=10):
+                mod = 0.25
+            elif ((elapsed_time > datetime.timedelta(minutes=10)) and (elapsed_time <= datetime.timedelta(minutes=30))):
```

Comment:
the first half of the check here and every subsequent elif check of the previous entry is not needed, if we are here we have already failed all previous checks so we know it's been over 10 minutes, therefore each time you only need the time you are actually checking. also the last elif check can simply be an `else:`